src/app/feedback/feedback.component.html

```diff
@@ -42,19 +42,25 @@
         </mat-cell>
       </ng-container>
       <!-- Priorty Column -->
-      <ng-container matColumnDef="priority">
+      <ng-container matColumnDef="priority" [ngSwitch]="true">
```

Comment:
ngswitch use should be based on variable and you do not need to do == case check on ngswitchcase

Please add ng-container inside mat-cell 
ngswitch=element.priority on ng-container
ngswitchcase=Yes on mat-chip-list